### PR TITLE
Add a convenience login script for a cloud workstation

### PR DESCRIPTION
**Pull request by geetduggal**
Pretty simple, but saves some time.  @jdidion @singerma @commandlinegirl interest

### CODE REVIEW

**Comment by commandlinegirl** [comment]
That's useful! Thanks, @geetduggal!

**Comment by ptn24** [comment]
How does this work? Just curious because I use cloud workstation at times

**Comment by Damien-Black** [comment]
This is useful:

1. `unset DX_WORKSPACE_ID` allows you to change the project (`dx select` any project) without `dx-toolkit` reverting back to the `DX_WORKSPACE_ID`.
    * Before if you relied on relative paths when using `dx` commands it would alway change you current project back to `DX_WORKSPACE_ID`
2. `dx cd $DX_PROJECT_CONTEXT_ID:` changes project context to the root of the project that launched the cloud workstation job
3. `sudo PYTHONPATH=$PYTHONPATH -E bash` Puts you in a subshell with root access and configures `PYTHONPATH` so that `dxpy` still works as expected. This mimics how our worker run job scripts.


Basically if you run this script, you'll be able to interactively work from a cloud workstation instance:
* using relative paths. And not being surprised when you upload a file to the jobs container instead of the parent project.
* Without specifying sudo for almost every command

**Comment by kpjensen** [comment]
@geetduggal since this is a bash script, not sure if we want to house it under `src/python`. What about under `bin/`?

File: src/python/scripts/dx-su-contrib [hunk 1]
```diff
@@ -0,0 +1,3 @@
+unset DX_WORKSPACE_ID
```
Comment (kpjensen) [inline src/python/scripts/dx-su-contrib]: `#!/bin/bash` shebang line?

Comment (geetduggal) [reply src/python/scripts/dx-su-contrib]: Yes, thanks!

**Comment by geetduggal** [comment]
> @geetduggal since this is a bash script, not sure if we want to house it under `src/python`. What about under `bin/`?

Done

**Comment by jdidion** [comment]
This is awesome @geetduggal!
